setup.cfg

```diff
@@ -1,55 +1,4 @@
-[metadata]
-name = aiohttp
-version = attr: aiohttp.__version__
-url = https://github.com/aio-libs/aiohttp
-project_urls =
-  Chat: Matrix = https://matrix.to/#/#aio-libs:matrix.org
-  Chat: Matrix Space = https://matrix.to/#/#aio-libs-space:matrix.org
-  CI: GitHub Actions = https://github.com/aio-libs/aiohttp/actions?query=workflow%%3ACI
-  Coverage: codecov = https://codecov.io/github/aio-libs/aiohttp
-  Docs: Changelog = https://docs.aiohttp.org/en/stable/changes.html
-  Docs: RTD = https://docs.aiohttp.org
-  GitHub: issues = https://github.com/aio-libs/aiohttp/issues
-  GitHub: repo = https://github.com/aio-libs/aiohttp
-description = Async http client/server framework (asyncio)
-long_description = file: README.rst
-long_description_content_type = text/x-rst
-maintainer = aiohttp team <team@aiohttp.org>
-maintainer_email = team@aiohttp.org
-license = Apache-2.0
-license_files = LICENSE.txt
-classifiers =
-  Development Status :: 5 - Production/Stable
-
-  Framework :: AsyncIO
-
-  Intended Audience :: Developers
-
-  License :: OSI Approved :: Apache Software License
-
-  Operating System :: POSIX
-  Operating System :: MacOS :: MacOS X
-  Operating System :: Microsoft :: Windows
-
-  Programming Language :: Python
-  Programming Language :: Python :: 3
-  Programming Language :: Python :: 3.9
-  Programming Language :: Python :: 3.10
-  Programming Language :: Python :: 3.11
-  Programming Language :: Python :: 3.12
-  Programming Language :: Python :: 3.13
-
-  Topic :: Internet :: WWW/HTTP
-
 [options]
-python_requires = >=3.9
-packages =
-  aiohttp
-  aiohttp._websocket
-# https://setuptools.readthedocs.io/en/latest/setuptools.html#setting-the-zip-safe-flag
-zip_safe = False
-include_package_data = True
-
 install_requires =
```

Comment:
I would have thought the dependencies would go in the same file as the rest of the metadata..?

Reply:
Yes they can be moved as well. Just the current tooling expects them to be in `setup.cfg`. Since this PR stalled for so long I figured I'd revert that part for now. See my last commit. https://github.com/aio-libs/aiohttp/pull/9951/commits/ec9022547a07f710a83a5c3e172884ccc82632bb

_I'd be happy to re-include it here or do it in a followup if there is interest._

Reply:
Right. I'll defer to @webknjaz here to decide how it should go.